_posts/briefs/2020-06-22-face-recog-police.md

```diff
@@ -0,0 +1,120 @@
+
+**From the Source:**
+
+After the announcements from IBM, Microsoft, and Amazon, Microsoft’s president [repeated the call](https://www.cnn.com/2020/06/18/tech/brad-smith-microsoft-facial-recognition/index.html) for federal regulation of facial recognition technology:
+
+> "We need to start teasing this issue apart, to understand it better and move just beyond a binary conversation of: permit it or ban it," Smith said. "And think about: what is the right way to regulate it?"
+
+**Summary**
+
+*   The enthusiasm for IBM’s, Microsoft’s, and Amazon’s moves on facial recognition have been mixed. 
+*   Adding to the worries are the existence of other companies peddling facial recognition technology, such as Clearview AI.
+*   While many see these announcements as a small step down a long road, they do see it as a step in the right direction.
+
+## **Our Perspective**
+
+Amazon and Microsoft have engaged in the same marketing of facial recognition that is being lambasted today. 
+While Amazon’s contracts are already public knowledge, Microsoft was recently found to have [pitched its facial recognition technology to the Drug Enforcement Administration](https://techcrunch.com/2020/06/17/microsoft-dea-facial-recognition/) (DEA) as far back as 2017. 
+The recent announcements from IBM, Microsoft, and Amazon are a promising step forward in the fight against unfettered use of facial recognition technology, particularly in policing. 
+There are strong arguments that due to the inherent difficulties in fairly implementing automation technology in law enforcement, [police should be banned from using facial recognition altogether](https://www.nytimes.com/2020/06/09/technology/facial-recognition-software.html). 
+In addition, as [John Oliver](https://www.youtube.com/watch?v=jZjmlJPJgug) pointed out, development of facial recognition in and of itself has opened a Pandora’s Box that even some in Silicon Valley were too afraid to touch. 
+
+<iframe width="560" height="315" src="https://www.youtube.com/embed/jZjmlJPJgug" frameborder="0" allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+
+In particular, the following immediate issues remain:
+
+*   Amazon is still engaged with the police in surveillance applications - Ring allegedly [has partnered with over 1000 police departments in the US](https://www.cnet.com/features/rings-work-with-police-lacks-solid-evidence-of-reducing-crime/).
+*   Smaller companies (Clearview, etc.): Even if Microsoft and Amazon decide to go as far as IBM has in the moral sense, the fact remains that companies like Clearview AI and Banjo continue to offer facial recognition services. As long as there are people who want to continue to use facial recognition for any purpose, its development will go on.
```

Comment:
I missed this earlier but since we're not doing this on the other bullet points, maybe remove "smaller companies:" here?